app/server/package.json

```diff
@@ -28,6 +29,7 @@
         "fastify-cors": "^6.0.2",
         "fastify-multipart": "^5.3.1",
         "form-data": "^4.0.0",
+        "fs": "^0.0.1-security",
```

Comment:
Hmm, why did fs get added here, that doesn't seem correct?  fs should already be part of nodejs itself.  If you check, this package doesn't actually do anything. https://www.npmjs.com/package/fs

Be careful when installing packages, you don't want to accidentally install a virus :smile:

Reply:
fs should be removed.

Reply:
removed!